apps/backoffice-v2/src/lib/blocks/components/EditableDetails/interfaces.ts

```diff
@@ -10,6 +11,7 @@ export type IEditableDetailsDocument = {
 export interface IEditableDetails {
   data: Array<{
     title: string;
+    titleNode?: ReactNode;
```

Comment:
ReactNode contains 'string'
```suggestion
    title: ReactNode;
```